src/mlpack/core/optimizers/sgd/update_policies/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(SOURCES
   vanilla_update.hpp
   momentum_update.hpp
+  ada_grad_update.hpp
```

Comment:
I looks like we moved the file from ``/core/optimizers/sgd/update_policies/``to the adagrad folder, so we should remove the entry from the sgd policies CMakeLists.txt file.